shared/actions/teams.tsx

```diff
@@ -1524,8 +1507,6 @@ const teamsSaga = function*() {
   yield* Saga.chainAction2(TeamsGen.updateTopic, updateTopic, 'updateTopic')
   yield* Saga.chainAction2(TeamsGen.updateChannelName, updateChannelname, 'updateChannelname')
   yield* Saga.chainAction2(TeamsGen.deleteChannelConfirmed, deleteChannelConfirmed, 'deleteChannelConfirmed')
-  yield* Saga.chainAction2(TeamsGen.badgeAppForTeams, badgeAppForTeams, 'badgeAppForTeams')
-  yield* Saga.chainAction2(TeamsGen.badgeAppForTeams, badgeAppForTeams, 'badgeAppForTeams')
```

Comment:
Unfortunate that there were two of these. Anyway this intermediate action wasn't needed, removed